src/app/common/validators/latin-letters-and-numbers-validator.ts

```diff
@@ -0,0 +1,13 @@
+import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
+
+export const LATIN_LETTERS_AND_NUMBERS_REGEXP = /^$|^[a-z0-9- ]+$/i;
+
+export function latinLettersAndNumbersValidator(): ValidatorFn {
```

Comment:
Maybe it is better to call this validator depends on whish fields this function is applied?
For example - authorFieldValidator().

Reply:
renamed